ome/omero-figure
# Xywh width height button

This PR adds an extra button to control the aspect ratio of the panels, when the user edits the width/height of the image.

To Test:

Test if toggling the glyphicon-link button changes status of glyphicon-ok-sign to show/hide.
When the glyphicon-ok-sign is hidden, edit width or height of the image. This will just change the width of height of the image.
When the glyphicon-ok-sign is shown, edit width or height of the image. Please check the width/height ratio (aspect ratio) of the panel before editing. Post editing the width/height of the panel, the height/width of the image should automatically get calculated and adjusted. The resultant panel aspect ratio, should be exactly the same as the previous aspect ratio.
Check if the tooltip on the button makes sense.
And as always code review would be useful as well.
The branch includes the code-layout branch as well, and the PR could technically be considered as an independent review of the code-layout PR(#178) from @will-moore
@will-moore @jburel

Conversation trace from closed PR: https://github.com/ome/omero-figure/pull/187
